tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CONST.py

```diff
@@ -2,52 +2,41 @@
 CONST file for CspSubarrayLeafNode
 """
 # In/Out command constants
-CMD_ASSIGN_RESOURCES = "AssignResources"
-CMD_RELEASE_ALL_RESOURCES = "ReleaseAllResources"
 CMD_ADD_RECEPTORS = "AddReceptors"
 CMD_REMOVE_ALL_RECEPTORS = "RemoveAllReceptors"
+CMD_CONFIGURESCAN = "ConfigureScan"
 
 #Event messages
 EVT_SUBSR_SA_RECEPTOR_ID_LIST = "receptorIDList"
 
-
 #Error messages
 ERR_IN_CREATE_PROXY = "Error in creating proxy of the LeafNode device: "
-ERR_INVALID_JSON = "Invalid JSON format"
-ERR_JSON_KEY_NOT_FOUND = "JSON key not found "
-ERR_RELEASE_ALL_RESOURCES_CMD = "Error occured while releasing resources "
-ERR_SUBARRAY_HEALTHSTATE = "Key Error occurred while setting Subarray healthState"
-ERR_ASSGN_RESOURCES = "Error occurred while assigning resources to the Subarray \n"
-ERR_RELEASE_RESOURCES = "Error occurred while releasing resources from the Subarray \n"
-ERR_STOW_ARGIN = "Invalid StowAntennas arguments \n"
+ERR_INVALID_JSON = "Invalid JSON format while invoking command on CspSubarray."
+ERR_JSON_KEY_NOT_FOUND = "JSON key not found."
+ERR_ASSGN_RESOURCES = "Error occurred while assigning resources to the Csp Subarray \n"
 ERR_INVOKING_CMD = "Error in invoking command: "
 ERR_EXCEPT_CMD_CB = "Exception in CommandCallback: \n"
 ERR_INIT_PROP_ATTR_CSPSALN = "Error on initialising properties and attributes " \
                         "on CspSubarrayLeaf Node device."
-ERR_IN_CREATE_PROXY = "Error in creating proxy of the device "
 ERR_MSG = "Error message is: "
-ERR_IN_CREATE_PROXY_CSPSA = "Error in creating proxy to the CSP Subarray "
 ERR_RELEASE_ALL_RESOURCES = "Error while invoking ReleaseAllResources command on CSP Subarray."
+ERR_CONFIGURESCAN_RESOURCES = "Error while invoking ConfigureScan command on CSP Subarray."
+ERR_INVALID_JSON_CONFIG_SCAN = "Invalid JSON format while invoking ConfigureScan command on CspSubarray."
+ERR_INVALID_JSON_ASSIGN_RES = "Invalid JSON format while invoking AddReceptors command on CspSubarray."
 
 #strings
 #General strings
 STR_RECEPTORID_LIST = "receptorIDList"
 STR_DISH = "dish"
-STR_HEALTH_STATE = "healthState of "
-STR_OK = " :-> OK"
-STR_DEGRADED = " :-> DEGRADED"
-STR_FAILED = " :-> FAILED"
-STR_UNKNOWN = " :-> UNKNOWN"
 STR_ERR_MSG = "Error message is: "
-STR_INIT_SUCCESS = "CspSubarrayLeafNode is initialized successfully."
-STR_FALSE = "False"
-STR_ASSIGN_RESOURCES_SUCCESS = "Resources are assigned successfully"
-STR_RELEASE_ALL_RESOURCES_SUCCESS = "All resources are removed successfully"
-STR_CSP_CBF_DEV_NAME = "mid_csp_cbf/sub_elt/master"
+STR_ASSIGN_RESOURCES_SUCCESS = "Resources are assigned successfully on CspSubarray."
+STR_RELEASE_ALL_RESOURCES_SUCCESS = "All resources assigned to CSP Subarray are removed successfully"
```

Comment:
Can change it to STR_REMOVE_ALL_RECEPTORS_SUCCESS.

Reply:
Changed STR_RELEASE_ALL_RESOURCES_SUCCESS to STR_REMOVE_ALL_RECEPTORS_SUCCESS in CONST file of CspSubarrayLeafNode.
Commit Id: [40fecec]